syftlib/src/main/java/org/openmined/syft/networking/requests/HttpAPI.kt

```diff
@@ -73,12 +75,27 @@ interface HttpAPI : CommunicationAPI {
         @Query("model_id") modelId: String
     ): Single<Response<ResponseBody>>
 
+    /**
+     * Calls `federated/authenticate` for authentication.
+     *
+     * @param authRequest Contains auth-token.
```

Comment:
Auth-token is the JWT authentication token responsible. Add description about JWT from readme